recipes/vtk/all/conandata.yml

```diff
@@ -0,0 +1,46 @@
+sources:
+  "9.2.6":
+    url: "https://www.vtk.org/files/release/9.2/VTK-9.2.6.tar.gz"
+    sha256: "06fc8d49c4e56f498c40fcb38a563ed8d4ec31358d0101e8988f0bb4d539dd12"
+
+patches:
```

Comment:
Some `patch_types` are not fitting. Some are rather `portability` patches, like the ones that enable newer C++ standards.
And for the one that is upstreamed please add a `patch_source`, it is much easier to know then what can be removed with future upstream releases

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patch_type

Reply:
Yep, was going to do that later as the upstreamed patches were upstreamed quite a while ago, and at the time hadn't been accepted yet.